doc/src/modules/physics/mechanics/joints.rst

```diff
@@ -0,0 +1,64 @@
+.. _joints_framework:
+
+=====================================
+Joints Framework in Physics/Mechanics
+=====================================
+
+:mod:`sympy.physics.mechanics` provides a joints framework. This system consists of two parts. The first are the :obj:`joints<sympy.physics.mechanics.joint>` themselves, which are used to create connections between :class:`bodies<sympy.physics.mechanics.body.Body>`. The second part is the :class:`~.JointsMethod`, which is used to form the equations of motion. Both of these parts are doing what we can call "book-keeping": keeping track of the relationships between :class:`bodies<sympy.physics.mechanics.body.Body>`.
+
+Joints in Physics/Mechanics
+===========================
+
+The general task of the :mod:`joints<sympy.physics.mechanics.joint>` is creating kinematic relationships between :class:`bodies<sympy.physics.mechanics.body.Body>`. Each joint has a setup as shown in the image below (this is the example of the :class:`~.PrismaticJoint`).
+
+.. image:: api/PrismaticJoint.*
+   :align: center
+   :width: 600
+
+As can be seen in this image, each joint needs several objects in order to define the relationships.
+First of it needs two bodies: the parent body (shown in green) and the child body (shown in blue).
+Both of these bodies have a masscenter from which the position of the joint is defined.
+In the parent body the vector from the mass center to the ``parent_point`` is called the ``parent_joint_pos``.
+For the child body these are called the ``child_point`` and ``child_joint_pos``.
+The orientation of the joint in each body is defined by the ``parent_axis`` and ``child_axis``.
+These two vectors are aligned as explained in the :class:`~.Joint` notes and are in the image parallel to the orange vector.
+As last the joint also needs :func:`~.dynamicsymbols` as generalized coordinates and speeds.
```

Comment:
I guess people could create these joints to just get static connections of bodies, so having a dynamicsymbol isn't strictly needed.